### PR TITLE
Handle raw identifiers

### PR DESCRIPTION
**Pull request by davidpdrsn**
If you have a schema like

```rust
#[juniper::object]
impl Query {
    fn r#type(r#fn: MyInputType) -> Vec<String> {
        unimplemented!()
    }
}

#[derive(GraphQLInputObject, Debug, PartialEq)]
struct MyInputType {
    r#trait: String,
}
```

The names in the GraphQL schema would still include `r#`. This patch fixes that for field names on fields of types, fields of input types, and arguments. Any other places where this should be fixed?

I discovered this while working on https://github.com/davidpdrsn/juniper-from-schema/pull/93.

### CODE REVIEW

File: juniper_codegen/src/util.rs [hunk 1]
```diff
@@ -593,6 +593,11 @@ pub struct GraphQLTypeDefinitionField {
     pub resolver_code: proc_macro2::TokenStream,
 }
 
+pub fn unraw(s: &str) -> String {
```
Comment (theduke) [inline juniper_codegen/src/util.rs]: I don't think we really need this helper.

`arg.name.unraw().to_string()` would be the same.

Comment (davidpdrsn) [reply juniper_codegen/src/util.rs]: [`unraw`](https://docs.rs/syn/1.0.7/syn/ext/trait.IdentExt.html) only works on `Ident` and the names here are `String`s. So it would become `format_ident!("{}", field.name).unraw().to_string();` which is a bit wordy. I don't mind changing it if you still think the helper is unnecessary.

**Comment by davidpdrsn** [comment]
Docs updated in b93fa61e9718a

File: juniper_codegen/src/util.rs (hunk 1 above)
Comment (theduke) [reply juniper_codegen/src/util.rs]: Ah, I missed that it was taking a `&str`. I think we will want some general refactoring of the macro code to keep the `Ident`s around to provide better error messages.

But we can do that in future work.

**Comment by theduke** [comment]
Squashed and merged, thanks!
